template/src/main/scala/ch.epfl.scala.index.views/package.scala

```diff
@@ -5,7 +5,7 @@ import model._
 
 package object html {
   // https://www.reddit.com/r/scala/comments/4n73zz/scala_puzzle_gooooooogle_pagination/d41jor5
-  def paginationRender(selected: Int, max: Int, toShow: Int = 10): (Option[Int], List[Int], Option[Int]) = {
+  def paginationRender(selected: Int, max: Int, toShow: Int = 20): (Option[Int], List[Int], Option[Int]) = {
```

Comment:
Doesn't showing more results per page decrease the amount of links shown? I don't get what you mean.